homeassistant/components/fritzbox/__init__.py

```diff
@@ -8,23 +8,31 @@
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
-    ATTR_DEVICE_CLASS,
-    ATTR_ENTITY_ID,
-    ATTR_NAME,
-    ATTR_UNIT_OF_MEASUREMENT,
```

Comment:
Why? Keep them.

Reply:
If it will change, the CI will catch it.

Reply:
ok, that convinces me :smiley: